Resources/Prototypes/Entities/Structures/Machines/material_reclaimer.yml

```diff
@@ -77,6 +78,8 @@
       - Brain
       tags:
       - HighRiskItem
+      - Bucket # Frontier
+      - GlassBeaker # Frontier
```

Comment:
Might look into why this processes syringes when empty but not full - could be another approach for this.  Ideally, I think unused, empty containers could be recycled, particularly jugs.